src/swsssdk/dbconnector.py

```diff
@@ -9,10 +9,61 @@
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
 class SonicDBConfig(object):
+    SONIC_DB_GLOBAL_CONFIG_FILE = "/var/run/redis/sonic-db/database_global.json"
     SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_dir = "/var/run/redis/sonic-db"
+    _sonic_db_global_config_init = False
     _sonic_db_config_init = False
     _sonic_db_config = {}
 
+    """This is the database_global.json parse and load API. This file has the namespace name and
+       the corresponding database_config.json file. The global file is significant for the
+       applications running in the linux host namespace, like eg: config/show cli, snmp etc which
+       needs to connect to databases running in other namesacpes. If the "namespace" attribute is not
+       specified for an "include" attribute, it referes to the linux host namespace.
+    """
+    @staticmethod
+    def load_sonic_global_db_config(global_db_file_path=SONIC_DB_GLOBAL_CONFIG_FILE):
+        """
+        Parse and load the global database config json file
+        """
+        if SonicDBConfig._sonic_db_global_config_init == True:
+            return
+
+        if os.path.isfile(global_db_file_path) ==  True:
+            global_db_config_dir = os.path.dirname(global_db_file_path)
+            with open(global_db_file_path, "r") as read_file:
+                all_ns_dbs = json.load(read_file)
+                for entry in all_ns_dbs['INCLUDES']:
+                    if 'namespace' not in entry.keys():
+                        # If the user already invoked load_sonic_db_config() explicitly to load the 
+                        # database_config.json file for current namesapce, skip loading the file 
+                        # referenced here in the global config file.
+                        if SonicDBConfig._sonic_db_config_init == True:
+                            continue
+                        ns = ''
+                    else: 
+                        ns = entry['namespace']
+
+                    # Check if _sonic_db_config already have this namespace present
+                    if ns in SonicDBConfig._sonic_db_config:
+                        msg = "The database_config for this namespace '{}' is already parsed. !!".format(ns)
+                        logger.warning(msg)
+                        continue
+
+                    db_include_file = os.path.join(global_db_config_dir, entry['include'])
+    
+                    # Not finding the database_config.json file for the namespace
+                    if os.path.isfile(db_include_file) == False:
+                        msg = "'{}' file is not found !!".format(db_include_file)
+                        logger.warning(msg)
+                        continue
+
+                    with open(db_include_file, "r") as inc_file:
+                        SonicDBConfig._sonic_db_config[ns] = json.load(inc_file)
```

Comment:
if ns is ' ', better to set _sonic_db_config_init to avoid duplicated load.

Reply:
same as earlier comment , to keep these two independent, the load_global_config and local_db_config